modules/fetchFishes.js

```diff
@@ -0,0 +1,36 @@
+import generateMarkup from './generateMarkup.js';
+import fetchLikes from './fetchLikes.js';
+
+const appId = 'daS11VuHj0e3k7bb2TZc';
```

Comment:
@jerryowusu , thank you for the review but I think Putting the appId in `.env` is not part of the requirements. Moreover doing this will mean it wont be accessible to github pages for hosting.